### PR TITLE
Modules update incl. fixing pigz in Pyrodigal

### LINKED ISSUE #291: Pigz error with pyrodigal
### Description of the bug

We have 2 user reports about failing pipeline runs when using prodigal for annotation:
- https://nfcore.slack.com/archives/C02K5GX2W93/p1684999768853429 (reported by @mathavanpu)
- https://nfcore.slack.com/archives/C02K5GX2W93/p1688471831570479 (reported by @avantonder)

Apparently, pigz as part of the pyrodigal module cannot process certain samples, see screenshot below.

### Command used and terminal output

```console
nextflow run nf-core/funcscan -profile singularity -c cambridge.config --input samplesheet.csv --run_amp_screening --run_arg_screening --amp_skip_hmmsearch -resume --outdir funcscan_results

(See error screenshot below)
```

### Relevant files

Error screenshot from @avantonder:
funcscan_error

### System information

_No response_

### PR DESCRIPTION
**Pull request by jasmezz**
Pull some updated modules.
Pyrodigal: Fix pigz symlink issue, closes #291 
GECCO: Due to renamed column in output file, comBGC had to be adjusted.

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/funcscan/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/funcscan _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,16 +8,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 
 - [#296](https://github.com/nf-core/funcscan/pull/296) Fixed empty output when saving prodigal annotations. (reported by @louperelo, fix by @jasmezz)
 - [#297](https://github.com/nf-core/funcscan/pull/297) Added check for empty annotation files prior going into screening. (❤️ to @alexhbnr for requesting, added by @jfy133)
+- [#299](https://github.com/nf-core/funcscan/pull/299) Fixed pigz error with symlinks in Pyrodigal, updated other modules: comBGC, FastQC, GECCO, gunzip, untar. (by @jasmezz)
 
 ### `Dependencies`
 
+| Tool  | Previous version | New version |
+| ----- | ---------------- | ----------- |
+| GECCO | 0.9.2            | 0.9.8       |
```
Comment (jfy133) [inline CHANGELOG.md:24]: Where are the version updates for the other tools? Have you validated the working correctly?

Comment (jasmezz) [reply CHANGELOG.md]: Added comBGC. :+1: The other modules (fastqc, gunzip, pyrodigal, untar) are just changes in the nf-core module, not actually version updates. And yes, all modules work correctly.

**Comment by jasmezz** [comment]
DeepARG workflow fails because [deepARG download server](http://bench.cs.vt.edu/deeparg) is down again :neutral_face:

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,16 +8,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
-- [#285](https://github.com/nf-core/funcscan/pull/285) Use nf-validation for samplesheet checking and added support for `fna.gz` input FASTA files (by @louperelo, @mirpedrol, @jfy133)
-- [#295](https://github.com/nf-core/funcscan/pull/295) Add Prokka to MultiQC output (by @louperelo)
+- [#285](https://github.com/nf-core/funcscan/pull/285) Use nf-validation for samplesheet checking and added support for `fna.gz` input FASTA files. (by @louperelo, @mirpedrol, @jfy133)
+- [#295](https://github.com/nf-core/funcscan/pull/295) Add Prokka to MultiQC output. (by @louperelo)
 
 ### `Fixed`
 
 - [#296](https://github.com/nf-core/funcscan/pull/296) Fixed empty output when saving prodigal annotations. (reported by @louperelo, fix by @jasmezz)
 - [#297](https://github.com/nf-core/funcscan/pull/297) Added check for empty annotation files prior going into screening. (❤️ to @alexhbnr for requesting, added by @jfy133)
+- [#299](https://github.com/nf-core/funcscan/pull/299) Fixed pigz error with symlinks in Pyrodigal, updated other modules: comBGC, FastQC, GECCO, gunzip, untar. (by @jasmezz)
```
Comment (jfy133) [inline CHANGELOG.md:18]:
```suggestion
- [#299](https://github.com/nf-core/funcscan/pull/299) Fixed pigz error with symlinks in Pyrodigal. (by @jasmezz)
```
just version bumps are fine just in dependencies, as there is no 'functional' change to the execution of the tools

**Review by jfy133 — APPROVED** [review]

**Pull request merged**